src/components/Quote/style.css

```diff
@@ -1,28 +1,28 @@
 .Quote {
-  @apply bg-gray-200;
+  @apply ui-bg-gray-200;
   .direction {
-    @apply p-4 flex my-4;
+    @apply ui-p-4 ui-flex ui-my-4;
   }
   .direction--left {
-    @apply flex-row;
+    @apply ui-flex-row;
     .quote {
       &:before {
-        @apply font-awesome;
+        @apply ui-font-awesome;
```

Comment:
I am not sure why, but in the component the quotes are not displayed.

Reply:
fixed